jobbergate-api/jobbergate_api/apps/job_scripts/routers.py

```diff
@@ -176,6 +181,76 @@ async def job_script_get(job_script_id: int = Query(...)):
     return response
 
 
+@router.patch(
+    "/job-scripts/{job_script_id}/upload",
+    status_code=status.HTTP_204_NO_CONTENT,
+    description="Endpoint to replace a job script file.",
+    dependencies=[Depends(guard.lockdown(Permissions.JOB_SCRIPTS_EDIT))],
+)
+async def job_script_replace_file_content(
```

Comment:
This doesn't change the "last modified" field for a job script. Could be an issue later.

Reply:
solved in [2e55bb4](https://github.com/omnivector-solutions/jobbergate/pull/257/commits/2e55bb49b2eccea7a199557ef4346a6c1d772a0b)